internal/jwt/standalone_issuer.go

```diff
@@ -0,0 +1,167 @@
+package jwt
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"sync"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+type StandaloneTokenIssuer struct {
+	clusterID types.NamespacedName
+	username  string
+	
+	currentToken   string
+	tokenMu        sync.RWMutex
+	rotationTicker *time.Ticker
+	stopCh         chan struct{}
+	
+	scontrolPath      string
+	rotationInterval  time.Duration
+}
+
+// NewStandaloneTokenIssuer creates a new standalone token issuer
+func NewStandaloneTokenIssuer(clusterID types.NamespacedName, username string) *StandaloneTokenIssuer {
+	return &StandaloneTokenIssuer{
+		clusterID:        clusterID,
+		username:         username,
+		scontrolPath:     "scontrol", // default path, can be overridden
+		rotationInterval: 30 * time.Minute, // rotate every 30 minutes
+	}
+}
+
+// WithScontrolPath sets a custom path for the scontrol command
+func (s *StandaloneTokenIssuer) WithScontrolPath(path string) *StandaloneTokenIssuer {
+	s.scontrolPath = path
+	return s
+}
+
+// WithRotationInterval sets a custom rotation interval
+func (s *StandaloneTokenIssuer) WithRotationInterval(interval time.Duration) *StandaloneTokenIssuer {
+	s.rotationInterval = interval
+	return s
+}
+
+// Start begins the token rotation process
+func (s *StandaloneTokenIssuer) Start(ctx context.Context) error {
+	// Generate initial token
+	if err := s.rotateToken(ctx); err != nil {
+		return fmt.Errorf("failed to generate initial Slurm token: %w", err)
+	}
+
+	// Start rotation ticker
+	s.rotationTicker = time.NewTicker(s.rotationInterval)
+	s.stopCh = make(chan struct{})
+
+	go s.rotationLoop(ctx)
+
+	return nil
+}
+
+// Stop stops the token rotation process
+func (s *StandaloneTokenIssuer) Stop() {
+	if s.rotationTicker != nil {
+		s.rotationTicker.Stop()
+	}
+	if s.stopCh != nil {
+		close(s.stopCh)
+	}
+}
+
+// rotationLoop runs the token rotation in the background
```

Comment:
Why not use the existing flow?
* get tokens on demand
* cache them until their expiry

This simplifies the implementation by removing `rotationLoop`, `Start`, and `Stop` methods, along with `stopCh` and probably the mutex.